packages/performance/package.json

```diff
@@ -1,26 +1,27 @@
 {
-    "name": "@ni-private/performance",
-    "version": "1.0.0",
-    "private": true,
-    "type": "module",
-    "description": "A package to measure the performance of the Nimble Components",
-    "scripts": {
-        "build": "tsc && vite build src",
-        "pack": "npm pack",
-        "lint": "eslint .",
-        "start": "vite src",
-        "performance": "lhci autorun"
-    },
-    "dependencies": {
-        "@ni/nimble-components": "*"
-    },
-    "devDependencies": {
-        "@lhci/cli": "^0.13.0",
-        "typescript": "~5.1.6",
-        "vite": "^5.1.5"
-    },
-    "files": [
-        "dist",
-        "!*.tgz"
-    ]
+  "name": "@ni-private/performance",
```

Comment:
can you avoid the whitespace change in this file? We normally do 4 space indents so this doesn't look right switching to 2 space

Reply:
Actually, all of our package.json files use two-space indentations. I intentionally changed this one to match.

Reply:
Oh interesting, maybe npm is doing that when it updates them? If it's matching the others that works 👍